pymeasure/adapters/prologix.py

```diff
@@ -64,17 +68,28 @@ class PrologixAdapter(SerialAdapter):
 
     """
 
-    def __init__(self, port, address=None, rw_delay=None, serial_timeout=0.5,
+    def __init__(self, resource_name, address=None, rw_delay=0, serial_timeout=None,
                  preprocess_reply=None, **kwargs):
-        super().__init__(port, timeout=serial_timeout,
-                         preprocess_reply=preprocess_reply, **kwargs)
+        # for legacy rw_delay: prefer new style over old one.
+        if rw_delay:
+            warn("Use 'query_delay' instead.", FutureWarning)
+            kwargs['query_delay'] = rw_delay
+        if serial_timeout:
+            warn("Use 'timeout' in ms instead", FutureWarning)
+            kwargs['timeout'] = serial_timeout
```

Comment:
What about removing `serial_timeout` ? I think the `timeout` is already OK.
Are you worried about backward compatibility ? In such case, I would remove it since we are not yet in 1.0 version.

Reply:
Exactly for backwards compatibility.
I tried to keep everything working as before. Changes are marked as deprecated though.